pipelines/test/ldd-check.yaml

```diff
@@ -0,0 +1,59 @@
+name: ldd-check
+
+needs:
+  packages:
+    - busybox
+    - posix-libc-utils
+
+inputs:
+  files:
+    description: |
+      The files to run `ldd` on and check for missing deps.
+    required: true
+  verbose:
+    description: |
+      Should the full ldd output be shown
+    required: false
+    default: false
+
+pipeline:
+  - name: "run ldd on provided files"
```

Comment:
Can you document if the files should be provided in a YAML list, or a space-separated list?

Reply:
Just curious if we anticipate many files being in the list, might be easier to visualize if it's a yaml list. <- drive by commentary no need for any action just thinking out loud. But +1 to what the format is.

Reply:
https://github.com/chainguard-dev/melange/issues/1388

Reply:
updated doc string.